Makefile

```diff
@@ -2,13 +2,26 @@ service := secureopenbanking-uk-iam-initializer
 gcr-repo := sbat-gcr-develop
 binary-name := initialize
 
+define clone_config_tag
+	if [ -d "config-repo" ]; then rm config-repo -rf; fi
+	if [ -d "config" ]; then rm config -rf; fi
+    git clone --depth 1 --branch ${github-tag} git@github.com:SecureApiGateway/fr-platform-config.git config-repo
+	mkdir config
+	mv config-repo/config/* config
+	rm -rf config-repo
+endef
+
+
```

Comment:
can this be removed now?

Reply:
yeah I want the tag clone for future, but will add it into the script commented out